draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1831,7 +1831,9 @@ might block third-party cookies entirely by refusing to send Cookie header field
 process Set-Cookie header fields during third-party requests. They might take a less
 draconian approach by partitioning cookies based on the first-party context,
 sending one set of cookies to a given third party in one first-party context,
-and another to the same third party in another.
+and another to the same third party in another. Or they might even send or set
+some third-party cookies, but block others depending on user-agent cookie policy
+or user controls.
```

Comment:
> depending on user-agent cookie policy or user controls

Do we want to include "cookie attributes" as something that could be considered?

Reply:
I don't think it's strictly necessary - if a UA can define a cookie policy, attributes can be part of that policy (even if not explicitly stated in the rfc).